package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.37.0",
+  "version": "1.37.1",
```

Comment:
@brendanww For versioning going into develop we add a development version, in this case would look like 1.37.0-0.0.1 and inc version after testing in stage when publishing/releasing
If you could make that update that would be awesome. Thanks man!!

Reply:
done! thanks for the quick feedback